Lib/cu2qu/ufo.py

```diff
@@ -164,18 +168,24 @@ def _glyphs_to_quadratic(glyphs, max_err, reverse_direction, stats):
     for segments in segments_by_location:
         tag = segments[0][0]
         if not all(s[0] == tag for s in segments[1:]):
-            raise IncompatibleGlyphsError(*glyphs)
+            num_incompatible_segments += 1
+            continue
         if tag == 'curve':
             segments = _segments_to_quadratic(segments, max_err, stats)
             glyphs_modified = True
         new_segments_by_location.append(segments)
 
+    if num_incompatible_segments:
+        glyphs_error = True
+        logger.error("Glyph %s has %d incompatible segment(s)" % (
```

Comment:
let's output that `s` for the plural based on the actual number of incompatible segments (i.e. only if greater than 1)

Reply:
sure...